src/Http/Controllers/Configuration/UserController.php

```diff
@@ -181,13 +179,43 @@ public function postReassignuser(ReassignUser $request)
     }
 
     /**
-     * @param $user_id
+     * @param \Seat\Web\Http\Validation\NewIntelNote $request
+     * @param int                                    $user_id
      *
      * @return mixed
      */
-    public function editUserAccountStatus($user_id)
+    public function editUserAccountStatus(NewIntelNote $request, int $user_id)
     {
 
+        $user = $this->getUser($user_id);
+        $current_group = $user->group;
+
+        // Set default note
+        $note = collect(sprintf('This character has been reactivated by %s.',
+            auth()->user()->name));
```

Comment:
I am not 100% on board with this. Adding notes like this will reveal to a standard user who manipulated their account (as well as tell them who else is on the instance). I would prefer to not leak such information to users other than administrators. Instead, the security log should be used for account manipulation events (always), and the character note may be added as convenience without the user who modified it.

Reply:
i moved the notes to the sec.log and removed the notes.